### PR TITLE
checker: avoid to merge region with high qps

### PR DESCRIPTION
**Pull request by lhy1024**
Signed-off-by: lhy1024 <admin@liudos.us>

### What problem does this PR solve?

avoid to merge region with high qps

### Check List

Tests

- Unit test
- Integration test

### Release note

```release-note
checker: avoid to merge region with high qps
```

### CODE REVIEW

File: server/config/config.go [hunk 1]
```diff
@@ -634,6 +634,8 @@ type ScheduleConfig struct {
 	MaxMergeRegionKeys uint64 `toml:"max-merge-region-keys" json:"max-merge-region-keys"`
 	// SplitMergeInterval is the minimum interval time to permit merge after split.
 	SplitMergeInterval typeutil.Duration `toml:"split-merge-interval" json:"split-merge-interval"`
+	// SplitQPSThreshold is the qps threshold for split region.
+	SplitQPSThreshold uint64 `toml:"split-qps-threshold" json:"split-qps-threshold"`
```
Comment (nolouch) [inline server/config/config.go:638]: can we know this from TiKV? I think we can use `ConfigManager`.

Comment (lhy1024) [reply server/config/config.go:638]: How to use `ConfigManager`？And I think we can also get `StoreHeartbeatInterval` and `RegionHeartbeatInterval` from it.

**Review by disksing — APPROVED** [review]

**Comment by disksing** [comment]
I think we need some document update.

File: server/config/config.go (hunk 1 above)
Comment (rleungx) [reply server/config/config.go:638]: What if this threshold is not the same between PD and TiKV?

Comment (lhy1024) [reply server/config/config.go:638]: When the pd threshold is higher than the tikv threshold, some regions may not be merged as expected, and conversely, some regions that were just split by qps may be merged again, so we expect them to be the same, or not much different

**Comment by lhy1024** [comment]
Considering that qps will be used as the dimension for reading hotspot scheduling and that the merge checker will skip regions already identified as hotspots, this pr is unnecessary and I will close it.

**Pull request closed without merging**